### DISCUSSION TITLE
How to specify both list style and list text style

### DISCUSSION THREAD

**Discussion by QiangF** [kjambunathan/org-mode-ox-odt#205] [Q&A] [answered]
For exammple:

    #+attr_odt: :style "Caption" :style "OrgNumberedList"
    1. item 1
    2. item 2

Or:

    #+odt_extra_styles: <style:style style:name="References" style:family="paragraph" style:parent-style-name="Standard"
    #+odt_extra_styles:              style:list-style-name="OrgNumberedSquareBracketList" style:class="extra">
    #+odt_extra_styles:   <style:text-properties fo:font-size="10.5pt"
    #+odt_extra_styles:                          style:font-size-complex="10.5pt"
    #+odt_extra_styles:                          style:font-size-asian="10.5pt"/>
    #+odt_extra_styles: </style:style>

    #+attr_odt: :style "References"
    1. item 1
    2. item 2

**Comment by kjambunathan** [comment, marked as answer]
![Screenshot from 2022-08-04 08-10-31](https://user-images.githubusercontent.com/19869864/182751603-be2e57f7-9501-4b3c-9812-2ae6d1c3cb99.png)


```
#+ATTR_ODT: :target "extra_styles"
#+begin_src nxml
<style:style style:name="MyBulletStyle"
             style:family="text">
  <style:text-properties fo:color="#ff0000" />
</style:style>
<text:list-style style:name="MyNumberedList">
  <text:list-level-style-number text:level="1"
                                text:style-name="MyBulletStyle"
                                style:num-format="1"
                                style:num-prefix="["
                                style:num-suffix="]">
    <style:list-level-properties text:min-label-width="0.64cm"
                                 text:space-before="0.641cm" />
  </text:list-level-style-number>
  <text:list-level-style-number text:level="2"
                                style:num-format="1"
                                style:num-prefix="["
                                style:num-suffix="]">
    <style:list-level-properties text:min-label-width="0.635cm"
                                 text:space-before="1.27cm" />
  </text:list-level-style-number>
</text:list-style>
#+end_src

#+attr_odt: :p-style "Text_20_body_20_bold" :style "MyNumberedList"
1. item 1
2. item 2
```
